jabgui/src/main/java/org/jabref/gui/consistency/ConsistencyCheckDialogViewModel.java

```diff
@@ -93,8 +89,8 @@ public ObservableList<ConsistencyMessage> getTableData() {
         return tableData;
     }
 
-    public Set<String> getColumnNames() {
-        Set<String> result = LinkedHashSet.newLinkedHashSet(columnCount + EXTRA_COLUMNS_COUNT);
+    public List<String> getColumnNames() {
+        List<String> result = new ArrayList<>(allReportedFields.size() + 2);
```

Comment:
since we removed the constant, maybe we add a comment for the 2 (I dont remember the exact reason but inferring from the constant name)
```suggestion
        List<String> result = new ArrayList<>(allReportedFields.size() + 2); // there are two extra columns
```

Reply:
It are the next two lines. 😅.

The constant was confusing to me, because there was also no reasoning about that mahic number. 😅